### PR TITLE
Add comment field for schema.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
ONCALL-46

Require https://github.com/vesoft-inc/nebula-common/pull/475

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

**Review by panda-sheep — DISMISSED** [review]
Good job，well done！

File: src/meta/processors/indexMan/CreateTagIndexProcessor.cpp [hunk 1]
```diff
@@ -151,6 +151,9 @@ void CreateTagIndexProcessor::process(const cpp2::CreateTagIndexReq& req) {
     item.set_schema_id(schemaID);
     item.set_schema_name(tagName);
     item.set_fields(std::move(columns));
+    if (req.__isset.comment) {
```
Comment (darionyaphet) [inline src/meta/processors/indexMan/CreateTagIndexProcessor.cpp:154]: This place should need to be modified

Comment (Shylock-Hg) [reply src/meta/processors/indexMan/CreateTagIndexProcessor.cpp]: Fixed.

**Review by yixinglu — APPROVED** [review]

**Review by critical27 — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]
Good job. Well done.

**Pull request merged**